src/middleware.js

```diff
@@ -38,22 +39,41 @@ function init(swaggerPath, options) {
                     schemas[parsedPath][currentMethod.toLowerCase()] = {};
 
                     const parameters = dereferenced.paths[currentPath][currentMethod].parameters || [];
-                    let bodySchema = parameters.filter(function (parameter) { return parameter.in === 'body' });
+
+                    let bodySchema = middlewareOptions.expectFormFieldsInBody
```

Comment:
Why should it be an option? you can either look for it all the time or understand it from the content-type

Reply:
If consumer is not using multer or custom middleware to put parsed fields into body, they won't be there by default.

Reply:
@idanto I can think of at least one case when that would be too limiting - some of our endpoints handle very large files, so in order to avoid processing them twice, we don't preprocess them with Multer, but write stream directly to s3. Considering that there are also form fields to be read, such approach would be impossible if preprocessing text fields would have been mandatory.

Reply:
so if I understand you correctly you, you ask for the ability to skip body validation in some cases, right?

If this is the case I will change the name of the option to something more general like `skipBodyValidation` and use it everywhere, even for JSON payload.
What do you think?

P.S - sorry for the late responses, let's try to close this one today.

Reply:
@idanto But the problem would be that you would disable the body validation for entire middleware in all of the application, which is suboptimal - if my endpoint is JSON-based, I definitely want body validated, I only don't want that to happen if I'm using a combination of form fields and files, and only if I don't preprocess them using Multer or something similar.

Reply:
but also in your case, you will disable it for all endpoints with the same behavior.
What about making this variable a list of endpoints to disable the body validation for them?

Reply:
@idanto That would mean that there is some centralized registry in application that is aware of specific endpoint and of their very precise implementation nuances, which is not a very good approach for a large, complex application, ideally endpoints should be self-contained.

More manageable approach would be implementing support for creating parametrized instances of validation middleware - currently swaggerValidator.validate method returns singleton that works the same for all endpoints, but there could be an alternative factory which would be accepting endpoint-specific parameters. This still is a very heavy-weight approach that would require more changes on the express-ajv-swagger-validation side, but it is doable.

What are your concerns over current implementation, which is as lightweight as possible in this case? If you dislike the opt-in behaviour of it, it could be opt-out instead, so that it expects form fields in the body by default, but may not be doing so if someone does not handle form fields that way.

Reply:
Much obliged!
BTW, express-ajv-swagger-validation feels very stable and feature-complete by now. What are your remaining plans till 1.0.0?

Reply:
let's put an issue for that and discuss it there. I have some few more things I would like to do before 1.0.0 like changing the loading and make sure we cover all swagger spec (or at least most of it) and also to check for openAPI 3.0 support